checkov/common/graph/graph_builder/graph_components/blocks.py

```diff
@@ -63,6 +63,10 @@ def get_attribute_dict(self, add_hash=True) -> Dict[str, Any]:
         base_attributes = self.get_base_attributes()
         self.get_origin_attributes(base_attributes)
 
+        if hasattr(self, "module_dependency") and hasattr(self, "module_dependency_num"):
+            base_attributes["module_dependency_"] = self.module_dependency
+            base_attributes["module_dependency_num_"] = self.module_dependency_num
```

Comment:
@gruebel Couldn't find it, but seems like `module_dependency_` and `module_dependency_num_` should be existing consts

Reply:
moved the attributes to the dataclass